nse_main.lua

```diff
@@ -1249,6 +1288,7 @@ if script_database_update then
   log_write("stdout", "Script Database updated successfully.");
 end
 
+assign_scripts()
```

Comment:
@dmiller-nmap With this call to `assign_scripts()`, `script_expression`(which holds the complete argument passed to script) gets split into commas  and added into rules. If I am not wrong then that's what you want to see in `get_chosen_scripts()`